src/main/java/com/cegeka/academy/web/rest/EventController.java

```diff
@@ -101,4 +107,16 @@ public void addUserToPublicEvent(@PathVariable(value = "id") Long id, @Valid @Re
 
     }
 
+    @GetMapping("/userEvents/{id}")
+    public List<EventDTO> getEventsByUser(@PathVariable(value = "id") Long id) throws NotFoundException {
+        return eventService.getEventsByUser(id);
+    }
+
+    @GetMapping("/byOwner/{id}")
```

Comment:
Regulile de pentru expunearea resurselor in REST sunt [aici](https://martinfowler.com/articles/richardsonMaturityModel.html); nivelul 3 nu intra in discutie.
Suntem pe resusrsa de evenimente, ca sa obtinem toate evenimentele adaugam */all* radacinii */event*.
Ca sa obtinem toate evenimentele pentru un anume owner adaugam */owner/{id}*.